interview-questions.md

```diff
@@ -7,51 +7,70 @@ First, without external resources. Challenge yourself to answer from memory as i
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
-
+test
 Your answer:
 
 Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: 
+//Git is used for moving, storing, updating version and sharing files. 
+
+//Git hub is a site that connects to the user(s) local computer allowing collaboration and another place to store code     
 
 Researched answer:
+//Git is a version control system, 
+
+//GitHub is a web-based platform for hosting Git repositories and collaborating on code
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: 
+//logical
 
 Researched answer:
+//Equality, Logical, Comparison
 
 1. What is the difference between an array's index and its value?
 
 Your answer:
+//an arrays index is a series of characters, value are the positions they're in 
 
 Researched answer:
+//The index is the position of an element in the array, while the value is the data stored at that position.
```

Comment:
Nice follow-up and correcting that answer. The index is the position of the element in the array, not the value.